packages/core/src/components/Label/Label.tsx

```diff
@@ -12,6 +12,14 @@ import useClickableProps from "../../hooks/useClickableProps/useClickableProps";
 import useMergeRef from "../../hooks/useMergeRef";
 import styles from "./Label.module.scss";
 import LabelCelebrationAnimation from "./LabelCelebrationAnimation";
+import { TextType } from "../Text/TextConstants";
+
+type Sizes = "small" | "medium";
```

Comment:
Let's move it to a new file called `Label.types.ts` (for refrences: `Search.types.ts`, `TextArea.types.ts` etc) as it's the new convention we're advancing